CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 9.0.0
```

Comment:
I don't know that a bump all the way up to version `9.0.0` is required. `8.13.0` seems more appropriate to me.